cnap/ui/src/store/index.ts

```diff
@@ -45,15 +48,29 @@ export const refreshPipeline = async (pipeline_db_url:string, ws_server_url:stri
     for (let pipeline of res.data) {
       urls.push(ws_server_url + "/" + pipeline.pipeline_id);
     }
+
+    let date = new Date(Date.parse(new Date().toString()));
+    let now_time = format_time(date);
+
     store.commit('updatePipelines',
       {'db_server': pipeline_db_url, 'ws_server': ws_server_url,
-        'pipelines': res.data, 'urls': urls});
+        'pipelines': res.data, 'urls': urls, 'now_time': now_time});
   } catch (error) {
     console.log(error);
     store.commit('cleanPipelines', {'db_server': pipeline_db_url, 'ws_server': ws_server_url});
   }
 }
 
+function format_time(date: Date) {
```

Comment:
Remove duplicate code, change to
function format_time(date: Date): string {
    const padZero = (num: number) => num < 10 ? `0${num}` : num.toString();

    const hour = padZero(date.getHours());
    const minute = padZero(date.getMinutes());
    const second = padZero(date.getSeconds());

    return `${hour}:${minute}:${second}`;
}

Reply:
Fixed, thanks.